runtime/src/main/scala/com/ing/baker/runtime/actor/serialization/modules/RecipeManagerModule.scala

```diff
@@ -1,15 +1,16 @@
 package com.ing.baker.runtime.actor.serialization.modules
 
 import com.ing.baker.il
+import com.ing.baker.runtime.actor.recipe_manager.protobuf.CompiledRecipeMap
 import com.ing.baker.runtime.actor.recipe_manager.{RecipeManager, RecipeManagerProtocol, protobuf}
 import com.ing.baker.runtime.actor.{recipe_manager, protobuf => ilp}
 
 class RecipeManagerModule extends ProtoEventAdapterModule {
 
   override def toProto(ctx: ProtoEventAdapter): PartialFunction[AnyRef, scalapb.GeneratedMessage] = {
-    case RecipeManager.RecipeAdded(recipeId, compiledRecipe) =>
+    case RecipeManager.RecipeAdded(recipeId, compiledRecipe, timeStamp) =>
       val compiledRecipeProto = ctx.toProto[ilp.CompiledRecipe](compiledRecipe)
-      protobuf.RecipeAdded(Option(recipeId), Option(compiledRecipeProto))
+      protobuf.RecipeAdded(Option(recipeId), Option(compiledRecipeProto), Option(timeStamp))
```

Comment:
Maybe better to use `Some(..)` here instead of `Option(..)`. This would actually hide an error, translating `null`, a non valid value to `None`, a valid protobuf field

Reply:
I aligned this with the rest of the code.
We can refactor this in another change all together.